test/aws_test.go

```diff
@@ -4,32 +4,244 @@
 package helmtest
 
 import (
-	"strconv"
+	"fmt"
 	"testing"
 
 	"github.com/gruntwork-io/terratest/modules/helm"
+	appsV1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	storageV1 "k8s.io/api/storage/v1"
 )
 
 func TestAWS(t *testing.T) {
-	objects, err := manifestToObjects(helm.RenderTemplate(t,
-		&helm.Options{
-			SetStrValues: map[string]string{
-				"pachd.storage.backend": "AMAZON",
-			},
-		}, "../pachyderm", "release-name", nil))
+
+	type envVarMap struct {
```

Comment:
Could definitely make this a package struct, was thinking it was a little clearer in line

Reply:
I think it's fine to declare it here.  You don't have to give it a name if you don't want to:

```go
testCases := []struct{
  helmKey, envVar, value string
}{
    {
        helmKey: "pachd.storage.amazon.bucket",
        envVar: "AMAZON_BUCKET",
        value: "my-bucket",
    },
    ...
}
```

Reply:
Will address in future PR